scripts/svm/acrossPlusJupiter.ts

```diff
@@ -0,0 +1,348 @@
+import { MulticallHandler } from "../../target/types/multicall_handler";
+import { formatUsdc, parseUsdc } from "./utils/helpers";
+import {
+  ASSOCIATED_TOKEN_PROGRAM_ID,
+  createApproveCheckedInstruction,
+  createAssociatedTokenAccountIdempotentInstruction,
+  createTransferCheckedInstruction,
+  getAssociatedTokenAddressSync,
+  getMinimumBalanceForRentExemptAccount,
+  getMint,
+  getOrCreateAssociatedTokenAccount,
+  TOKEN_PROGRAM_ID,
+} from "@solana/spl-token";
+import {
+  AcrossPlusMessageCoder,
+  calculateRelayHashUint8Array,
+  intToU8Array32,
+  loadFillV3RelayParams,
+  MulticallHandlerCoder,
+  prependComputeBudget,
+  sendTransactionWithLookupTable,
+  getSolanaChainId,
+  isSolanaDevnet,
+  SOLANA_SPOKE_STATE_SEED,
+  SOLANA_USDC_MAINNET,
+} from "../../src/svm";
+import { CHAIN_IDs } from "../../utils/constants";
+import { FillDataParams, FillDataValues } from "../../src/types/svm";
+
+const swapApiBaseUrl = "https://quote-api.jup.ag/v6/";
+
+// Set up Solana provider and signer.
+const provider = AnchorProvider.env();
+anchor.setProvider(provider);
+const relayer = (provider.wallet as Wallet).payer;
+
+// Get Solana programs.
+const svmSpokeIdl = require("../../target/idl/svm_spoke.json");
+const svmSpokeProgram = new Program<SvmSpoke>(svmSpokeIdl, provider);
+const handlerIdl = require("../../target/idl/multicall_handler.json");
+const handlerProgram = new Program<MulticallHandler>(handlerIdl, provider);
+
+if (isSolanaDevnet(provider)) throw new Error("This script is only for mainnet");
+
+// Parse arguments
+const argv = yargs(hideBin(process.argv))
+  .option("recipient", { type: "string", demandOption: true, describe: "Recipient public key" })
+  .option("outputMint", { type: "string", demandOption: true, describe: "Token to receive from the swap" })
+  .option("usdcValue", { type: "string", demandOption: true, describe: "USDC value bridged/swapped (formatted)" })
+  .option("slippageBps", { type: "number", demandOption: false, describe: "Custom slippage in bps" })
+  .option("maxAccounts", { type: "number", demandOption: false, describe: "Maximum swap accounts" })
+  .option("priorityFeePrice", { type: "number", demandOption: false, describe: "Priority fee price in micro lamports" })
+  .option("fillComputeUnit", { type: "number", demandOption: false, describe: "Compute unit limit in fill" }).argv;
+
+async function acrossPlusJupiter(): Promise<void> {
+  const resolvedArgv = await argv;
+  const seed = SOLANA_SPOKE_STATE_SEED; // Seed is always 0 for the state account PDA in public networks.
+  const recipient = new PublicKey(resolvedArgv.recipient);
+  const outputMint = new PublicKey(resolvedArgv.outputMint);
+  const usdcAmount = parseUsdc(resolvedArgv.usdcValue);
+  const slippageBps = resolvedArgv.slippageBps || 100; // default to 1%
+  const maxAccounts = resolvedArgv.maxAccounts || 24;
+  const priorityFeePrice = resolvedArgv.priorityFeePrice;
+  const fillComputeUnit = resolvedArgv.fillComputeUnit || 400_000;
```

Comment:
Is it possible to calculate this on the fly instead of hardcoding it or is it difficult to estimate the Jupiter transactions consumption?